### DISCUSSION TITLE
画布初始化数据后，画布方法不正常了，怎么解决？

### DISCUSSION THREAD

**Discussion by Tolerate-yu** [antvis/X6#4420] [Q&A]
![动画](https://github.com/user-attachments/assets/4eb91a64-16ca-4756-87bd-ee13672e58e6)
 this.graph.fromJSON(this.drawData)  使用这个方法初始化画布上的节点后，画布方法就不正常了，怎么解决

**Comment by zlecheng** [comment]
> ![动画](https://private-user-images.githubusercontent.com/103416625/363092172-4eb91a64-16ca-4756-87bd-ee13672e58e6.gif?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjUyNDUwMzIsIm5iZiI6MTcyNTI0NDczMiwicGF0aCI6Ii8xMDM0MTY2MjUvMzYzMDkyMTcyLTRlYjkxYTY0LTE2Y2EtNDc1Ni04N2JkLWVlMTM2NzJlNThlNi5naWY_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwOTAyJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDkwMlQwMjM4NTJaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1lM2QxZDkwYzVmZjdhZjkxMTI2NDlmMzlmMmQwZThlNDkwYjI5MTkzZjliOTliOGNlZWU3YjI3MjNjZGVjYzRlJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.o23WXiCef7hkDU3tZAzHQJqvFrp9lpW9i7OGUS6tc5E) [ ![动画](https://private-user-images.githubusercontent.com/103416625/363092172-4eb91a64-16ca-4756-87bd-ee13672e58e6.gif?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjUyNDUwMzIsIm5iZiI6MTcyNTI0NDczMiwicGF0aCI6Ii8xMDM0MTY2MjUvMzYzMDkyMTcyLTRlYjkxYTY0LTE2Y2EtNDc1Ni04N2JkLWVlMTM2NzJlNThlNi5naWY_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwOTAyJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDkwMlQwMjM4NTJaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1lM2QxZDkwYzVmZjdhZjkxMTI2NDlmMzlmMmQwZThlNDkwYjI5MTkzZjliOTliOGNlZWU3YjI3MjNjZGVjYzRlJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.o23WXiCef7hkDU3tZAzHQJqvFrp9lpW9i7OGUS6tc5E) ](https://private-user-images.githubusercontent.com/103416625/363092172-4eb91a64-16ca-4756-87bd-ee13672e58e6.gif?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjUyNDUwMzIsIm5iZiI6MTcyNTI0NDczMiwicGF0aCI6Ii8xMDM0MTY2MjUvMzYzMDkyMTcyLTRlYjkxYTY0LTE2Y2EtNDc1Ni04N2JkLWVlMTM2NzJlNThlNi5naWY_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwOTAyJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDkwMlQwMjM4NTJaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1lM2QxZDkwYzVmZjdhZjkxMTI2NDlmMzlmMmQwZThlNDkwYjI5MTkzZjliOTliOGNlZWU3YjI3MjNjZGVjYzRlJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.o23WXiCef7hkDU3tZAzHQJqvFrp9lpW9i7OGUS6tc5E) [ ](https://private-user-images.githubusercontent.com/103416625/363092172-4eb91a64-16ca-4756-87bd-ee13672e58e6.gif?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjUyNDUwMzIsIm5iZiI6MTcyNTI0NDczMiwicGF0aCI6Ii8xMDM0MTY2MjUvMzYzMDkyMTcyLTRlYjkxYTY0LTE2Y2EtNDc1Ni04N2JkLWVlMTM2NzJlNThlNi5naWY_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwOTAyJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDkwMlQwMjM4NTJaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1lM2QxZDkwYzVmZjdhZjkxMTI2NDlmMzlmMmQwZThlNDkwYjI5MTkzZjliOTliOGNlZWU3YjI3MjNjZGVjYzRlJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.o23WXiCef7hkDU3tZAzHQJqvFrp9lpW9i7OGUS6tc5E) this.graph.fromJSON(this.drawData) 使用这个方法初始化画布上的节点后，画布方法就不正常了，怎么解决

有demo？
